eox_nelp/user_profile/api/v1/views.py

```diff
@@ -133,3 +134,63 @@ def get_validated_user_fields(request):
     validated_fields = validate_required_user_fields(request.user)
 
     return Response(validated_fields, status=status.HTTP_200_OK)
+
+
+@api_view(["GET"])
+@authentication_classes((SessionAuthenticationAllowInactiveUser,))
+@permission_classes((IsAuthenticated,))
+def get_conditional_user_fields(request):  # pylint: disable=unused-argument
```

Comment:
Only a question. Why do we need this endpoint? We also going to have a  post method by tenant config for this method?. Or this is only to verify conditional-fields

Reply:
You made me wonder if we really need this method,  and now I think we don't need this, the idea is to share that configuration with the frontend  but we can do that with the MFE_CONFIG API

Reply:
Go with the mfe_config_map